StarryExpanse/SaveGame/rivensavegame_generator.py

```diff
@@ -1,91 +1,32 @@
 import cog
+import yaml
+
+root = None
+
+with open("./Config/save_game_structure.yaml", 'r') as stream:
+    try:
+        root = yaml.load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
```

Comment:
nit: could do print(exc, file=sys.stderr). The new buildbot build system will colorize stderr making it easier to find the cause of build errors.